### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/fa74071534dbacb4bced549ec5b8fa128b071a7d

### CODE REVIEW

File: en/products/mozsocial/invite.ftl [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+mozsocial-invite-welcome-to-mozillasocial = Welcome to { -brand-name-mozilla-social }!
+mozsocial-invite-please-join-our-waitlist = Please join our waitlist by completing the form below.
+mozsocial-invite-required-label = Required
+mozsocial-invite-first-name-label = First name
+mozsocial-invite-last-name-label = Last name
+mozsocial-invite-your-email-address-label = Your email address
+mozsocial-invite-country-label = Country
+
+# "Handle" is jargon for a user name.
+mozsocial-invite-twitter-handle-label = Twitter handle?
```
Comment (peiying2) [inline en/products/mozsocial/invite.ftl:18]: Shall we change to `X (formerly Twitter)`?

**Review by peiying2 — CHANGES_REQUESTED** [review]
The pontoon.toml file needs to include a new entry because of this newly added file.

Also while we are editing the pontoon.toml file, shall we simplify the entries for VPN pages: [.../products/vpn/*.ftl](https://github.com/mozilla-l10n/www-l10n/blob/aff668ed2047eb87efd9a6d490379df2e146a630/configs/pontoon.toml#L315 )?

File: en/firefox/new/desktop.ftl [hunk 2]
```diff
@@ -140,10 +140,15 @@ firefox-desktop-download-firefox-was-created = { -brand-name-firefox } was creat
...
 firefox-desktop-download-as-the-internet = As the internet grows and changes, { -brand-name-firefox } continues to focus on your right to privacy  — we call it the <a { $attrs }>Personal Data Promise</a>: Take less. Keep it safe. No secrets. Your data, your web activity, your life online is protected with { -brand-name-firefox }.
 
+# Obsolete string
 # Variables:
 #   $attrs (attrs) - link to https://accounts.firefox.com/signin
 firefox-desktop-download-firefox-is-available = { -brand-name-firefox } is available on all your devices; take your tabs, history and bookmarks with you. All you need is a <a { $attrs }>{ -brand-name-firefox } account</a>.
 
+# Variables:
+#   $attrs (attrs) - link to https://accounts.firefox.com/signin
+firefox-desktop-download-firefox-is-available-v2 = { -brand-name-firefox } is available on all your devices; take your tabs, history and bookmarks with you. All you need is a <a { $attrs }>{ -brand-name-mozilla } account</a> and you’ll get access to syncing and more { -brand-name-mozilla } products.
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:150]: Placeholder should include `account`

File: en/products/mozsocial/invite.ftl [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (peiying2) [inline en/products/mozsocial/invite.ftl:1]: Need to add this file as an entry to the Potoon.toml file

**Review by peiying2 — CHANGES_REQUESTED** [review]
A couple of more comments added.

**Comment by peiying2** [comment]
@flodolo let me know if you have time or I can ask Bryan to help.

File: en/products/mozsocial/invite.ftl (hunk 1 above)
Comment (peiying2) [reply en/products/mozsocial/invite.ftl:18]: The Mozilla Social file was a smaller PR that came earlier but was not addressed and fixed when the bigger PR got merged with the older request.

Comment (nathan-barrett) [reply en/products/mozsocial/invite.ftl:18]: I will make a fix for issues that came up with the Mozilla Social PR now, thanks @peiying2

File: en/firefox/privacy/passwords.ftl [hunk 4]
```diff
@@ -50,22 +50,36 @@ privacy-passwords-the-secret-to = The secret to preventing guessing, theft or pa
...
 
+
+# Obsolete string
 #   $url_sumo-manager (string) - link to https://support.mozilla.org/kb/password-manager-remember-delete-edit-logins with additional attributes for analytics
-privacy-passwords-when-youre-logged = When you’re logged into { -brand-name-firefox } with your { -brand-name-firefox } account, you can sync across all your devices and access your passwords from a { -brand-name-firefox } mobile browser. Learn more about <a { $url_sumo-manager }>how to use the built-in password manager</a> to the fullest here.
+privacy-passwords-when-youre-logged = When you’re logged into { -brand-name-firefox } with your { -brand-name-firefox } account, you can sync across all your devices and access your passwords from a { -brand-name-firefox } mobile browser. Learn more about <a { $url_sumo_manager }>how to use the built-in password manager</a> to the fullest here.
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply en/firefox/privacy/passwords.ftl:69]: Oh wait, the string below replaces it, so then it does make sense to revert this even though it was broken for a long time :/

**Review by peiying2 — APPROVED** [review]
Thanks for the fixes.

**Pull request merged**